keras/backend/theano_backend.py

```diff
@@ -202,6 +202,12 @@ def any(x, axis=None, keepdims=False):
     return T.any(x, axis=axis, keepdims=keepdims)
 
 
+def all(x, axis=None, keepdims=False):
```

Comment:
Similar issues between `any` and `all`, tensorflow doesn't support boolean to integer casting